<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>993</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_15">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_14">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QLabel" name="label_25">
            <property name="text">
             <string>GCode</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditGcode"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTextBrowser" name="textBrowserConsole"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <layout class="QGridLayout" name="gridLayout">
                <item row="4" column="0">
                 <layout class="QHBoxLayout" name="horizontalLayout_10">
                  <item>
                   <widget class="QLCDNumber" name="lcdNumberXPos">
                    <property name="value" stdset="0">
                     <double>0.000000000000000</double>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButtonXZero">
                    <property name="text">
                     <string>Zero</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="5" column="0">
                 <layout class="QGridLayout" name="gridLayout_4">
                  <item row="1" column="1">
                   <widget class="QLineEdit" name="lineEditXFeed"/>
                  </item>
                  <item row="3" column="0">
                   <widget class="QLabel" name="label_11">
                    <property name="text">
                     <string>Min</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="lineEditXTravel"/>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>Feed</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QLineEdit" name="lineEditXMax"/>
                  </item>
                  <item row="3" column="1">
                   <widget class="QLineEdit" name="lineEditXMin"/>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="label_10">
                    <property name="text">
                     <string>Max</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_2">
                    <property name="text">
                     <string>Travel</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>X Axis</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QPushButton" name="pushButtonXPlus">
                  <property name="text">
                   <string>+</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QPushButton" name="pushButtonXMinus">
                  <property name="text">
                   <string>-</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <property name="sizeConstraint">
                   <enum>QLayout::SetFixedSize</enum>
                  </property>
                  <item>
                   <widget class="QLabel" name="label_22">
                    <property name="text">
                     <string>Step Size</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="lineEditXStepSize"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QGridLayout" name="gridLayout_5">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_12">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Y Axis</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QPushButton" name="pushButtonYMinus">
                  <property name="text">
                   <string>-</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QPushButton" name="pushButtonYPlus">
                  <property name="text">
                   <string>+</string>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <layout class="QHBoxLayout" name="horizontalLayout_11">
                  <item>
                   <widget class="QLCDNumber" name="lcdNumberYPos"/>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButtonYZero">
                    <property name="text">
                     <string>Zero</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="5" column="0">
                 <layout class="QGridLayout" name="gridLayout_6">
                  <item row="3" column="0">
                   <widget class="QLabel" name="label_13">
                    <property name="text">
                     <string>Min</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="1">
                   <widget class="QLineEdit" name="lineEditYFeed"/>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="lineEditYTravel"/>
                  </item>
                  <item row="3" column="1">
                   <widget class="QLineEdit" name="lineEditYMin"/>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="label_15">
                    <property name="text">
                     <string>Max</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QLineEdit" name="lineEditYMax"/>
                  </item>
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_14">
                    <property name="text">
                     <string>Feed</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_16">
                    <property name="text">
                     <string>Travel</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="1" column="0">
                 <layout class="QHBoxLayout" name="horizontalLayout_8">
                  <property name="sizeConstraint">
                   <enum>QLayout::SetFixedSize</enum>
                  </property>
                  <item>
                   <widget class="QLabel" name="label_23">
                    <property name="text">
                     <string>Step Size</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="lineEditYStepSize"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QGridLayout" name="gridLayout_7">
                <item row="2" column="0">
                 <widget class="QPushButton" name="pushButtonZPlus">
                  <property name="text">
                   <string>+</string>
                  </property>
                 </widget>
                </item>
                <item row="3" column="0">
                 <widget class="QPushButton" name="pushButtonZMinus">
                  <property name="text">
                   <string>-</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_17">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text">
                   <string>Z Axis</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="4" column="0">
                 <layout class="QHBoxLayout" name="horizontalLayout_12">
                  <item>
                   <widget class="QLCDNumber" name="lcdNumberZPos"/>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButtonZZero">
                    <property name="text">
                     <string>Zero</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item row="5" column="0">
                 <layout class="QGridLayout" name="gridLayout_8">
                  <item row="1" column="0">
                   <widget class="QLabel" name="label_19">
                    <property name="text">
                     <string>Feed</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0">
                   <widget class="QLabel" name="label_20">
                    <property name="text">
                     <string>Max</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="0">
                   <widget class="QLabel" name="label_21">
                    <property name="text">
                     <string>Travel</string>
                    </property>
                   </widget>
                  </item>
                  <item row="3" column="0">
                   <widget class="QLabel" name="label_18">
                    <property name="text">
                     <string>Min</string>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="1">
                   <widget class="QLineEdit" name="lineEditZMax"/>
                  </item>
                  <item row="3" column="1">
                   <widget class="QLineEdit" name="lineEditZMin"/>
                  </item>
                  <item row="0" column="1">
                   <widget class="QLineEdit" name="lineEditZTravel"/>
                  </item>
                  <item row="1" column="1">
                   <widget class="QLineEdit" name="lineEditZFeed"/>
                  </item>
                 </layout>
                </item>
                <item row="1" column="0">
                 <layout class="QHBoxLayout" name="horizontalLayout_9">
                  <property name="sizeConstraint">
                   <enum>QLayout::SetFixedSize</enum>
                  </property>
                  <item>
                   <widget class="QLabel" name="label_24">
                    <property name="text">
                     <string>Step Size</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="lineEditZStepSize"/>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
